src/js/msp.js

```diff
@@ -147,13 +234,26 @@ var MSP = {
     clearListeners: function() {
         this.listeners = [];  
     },
-    send_message: function (code, data, callback_sent, callback_msp, doCallbackOnError) {
-        if (code === undefined) {
-            return;
+    crc8_dvb_s2: function(crc, ch) {
+        crc ^= ch;
+        for (let ii = 0; ii < 8; ++ii) {
```

Comment:
`ii` is not used at all at the loop, I think usually `ii++` is prettier and more habitual than `++i`

Reply:
`ii` is only used to count up to 8 iterations. If you prefer another style of looping, I can definitely change it. In the meantime, I've changed `++ii` to `ii++`.